request-handler/src/main/java/au/org/aodn/aws/wps/operation/ExecuteOperation.java

```diff
@@ -27,28 +34,43 @@ public ExecuteOperation(Execute executeRequest) {
     }
 
     @Override
-    public Object execute() {
-        //TODO: Read config
+    public Object execute(Properties config) {
+
+        //  Config items:
+        //      queue names
+        //      job name
+        //      AWS region
+        //      status filename
+        //      status location
+        String statusLocationBase = config.getProperty(STATUS_LOCATION_BASE_CONFIG_KEY);
```

Comment:
Would it  be better to make them "STATUS_S3_BUCKET_CONFIG_KEY" and "STATUS_S3_KEY_CONFIG_KEY" since that's what will be needed for an s3 upload?